projects/adf-office-services-ext/README.md

```diff
@@ -70,3 +70,9 @@ Update `app.extensions.json` and append a reference to the plugin definition:
   "$references": ["aos.plugin.json"]
 }
 ```
+
+## Disable and Enable the extension after it is installed
+
+There's an environment that can disable or enable the installed extension.
```

Comment:
Could we specify the default value is true, please?

Reply:
Yes, of course. Thanks for pointing that out.